spec/models/project_spec.rb

```diff
@@ -281,9 +281,9 @@
 
       ordered_projects = Project.by_provisional_conversion_date
 
-      expect(ordered_projects[0]).to eq first_project
-      expect(ordered_projects[1]).to eq middle_project
-      expect(ordered_projects[2]).to eq last_project
+      expect(ordered_projects[0].provisional_conversion_date).to eq first_project.provisional_conversion_date
```

Comment:
I would be tempted to use the `id` here as being closer to what we are trying to test?

Reply:
Good shout!